paper-input-behavior.html

```diff
@@ -384,7 +384,16 @@
         label.id = labelledBy;
       }
       this._ariaLabelledBy = labelledBy;
-    }
+    },
+
+    _onChange: function(event) {
+      // stop the propagation in shady DOM
+      event.stopPropagation();
```

Comment:
`paper-checkbox` and friends fire an analogous `iron-change` event when the `checked` value changes, so that could be a good name. `paper-checkbox` also fires the `change` event to emulate the native checkbox, so two events isn't unheard of, I don't think